packages/relay/src/lib/services/debugService/index.ts

```diff
@@ -0,0 +1,118 @@
+import { Logger } from 'pino';
+import { MirrorNodeClient } from '../../clients';
+import { IDebugService } from './IDebugService';
+import { CacheService } from '../cacheService/cacheService';
+import { predefined } from '../../errors/JsonRpcError';
+import { CommonService } from '../ethService/ethCommonService';
+
+export class DebugService implements IDebugService {
+  /**
+   * The interface through which we interact with the mirror node
+   * @private
+   */
+  private readonly mirrorNodeClient: MirrorNodeClient;
+
+  /**
+   * The logger used for logging all output from this class.
+   * @private
+   */
+  private readonly logger: Logger;
+
+  /**
+   * The LRU cache used for caching items from requests.
+   *
+   * @private
+   */
+  private readonly cacheService: CacheService;
+  private readonly common: CommonService;
+  public readonly debugTraceTransaction = 'debug_traceTransaction';
+
+  constructor(mirrorNodeClient: MirrorNodeClient, logger: Logger, cacheService: CacheService, common: CommonService) {
+    this.mirrorNodeClient = mirrorNodeClient;
+    this.logger = logger;
+    this.cacheService = cacheService;
+    this.common = common;
+  }
+
+  /**
+   * Checks if the Filter API is enabled
+   */
+  static requireDebugAPIEnabled(): void {
+    if (!process.env.DEBUG_API_ENABLED || process.env.DEBUG_API_ENABLED !== 'true') {
+      throw predefined.UNSUPPORTED_METHOD;
+    }
+  }
```

Comment:
I believe is better to read the contents of the `ENV` variable once and have something like this:

Bit more readable and condition is already processed.
```js
 /**
   * Checks if the Filter API is enabled
   * @public
   */
  public static readonly isDebugAPIEnabled = process.env.DEBUG_API_ENABLED === 'true' || false;
  /**
   * Checks if the Filter API is enabled
   */
  static requireDebugAPIEnabled(): void {
    if (!this.isDebugAPIEnabled) {
      throw predefined.UNSUPPORTED_METHOD;
    }
  }
```